cmd/sops/main.go

```diff
@@ -443,6 +444,55 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:      "filestatus",
+			Usage:     "check the status of the file, returning encryption status",
+			ArgsUsage: `file`,
+			Flags:     []cli.Flag{},
+			Action: func(c *cli.Context) error {
```

Comment:
Could you extract the body of this function to a package in the `subcommands` package? Like with https://github.com/mozilla/sops/blob/master/cmd/sops/subcommand/updatekeys/updatekeys.go and others. I think it makes the code easier to navigate vs having a lot of unrelated functionality in the same file.

Reply:
done in https://github.com/mozilla/sops/pull/545/commits/6e32720f45eb9a149f810540ee64e689d8662308